src/views/Simulation/Recapitulatif.vue

```diff
@@ -0,0 +1,156 @@
+<template>
+  <form @submit.prevent="onSubmit" class="recapitulatif-form">
+    <div>
+      <template v-for="(chapter, chapterIndex) in chapters">
+        <div
+          class="chapter-block"
+          :key="chapter.name"
+          v-if="chapter.questions.length"
+        >
+          <h2 class="aj-question">{{ chapter.label }}</h2>
+          <template v-for="(question, questionIndex) in chapter.questions">
+            <template v-if="question.isChapterSubtitle">
+              <div
+                :key="`chapter_${chapterIndex}_question_${questionIndex}`"
```

Comment:
having indexes as key is not a good practice :/
can't we use the labels ?

Reply:
Ok je ne savais pas que c'était une mauvaise pratique. :)
Je l'ai utilisé dans le cas où le for a deux div, dans ce cas l'utilisation du div n'est pas possible.